### DISCUSSION TITLE
狡兔屋正式入驻

### DISCUSSION THREAD

**Discussion by keerdun** [share121/inter-knot#887] [其他]
![IMG_20240810_181336](https://github.com/user-attachments/assets/6d5a6824-4628-4b3e-9210-31d98577f81d)
 大家好，我们狡兔屋事务所正式入驻绳网了，万能的事务所专接各种委托，服务质量上乘，坚决以委托人利益为核心
 励志接好每一单委托！

**Comment by sky27950** [comment]
法厄同阁下托我问你们妮可老大，什么时候还债？

**Comment by XiaoFffeng** [comment]
我是法厄同，什么时候还钱

**Comment by ZhuYuan-148** [comment]
久闻狡兔屋，什么时候来治安局坐坐？

**Comment by Bochiii984** [comment]
记得还债喵

**Comment by SSTV-10068** [comment]
下面还有一个狡兔屋，请问哪个才是真的！
